botx/models/chats.py

```diff
@@ -28,3 +29,47 @@ class ChatFromSearch(BotXBaseModel):
 
     #: users in chat.
     members: List[UserFromChatSearch]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+
+class BotChat(BotXBaseModel):
+    """Chat from list."""
+
+    #: name of chat.
+    name: str
+
+    #: description of chat
+    description: Optional[str]
+
+    #: type of chat.
+    chat_type: ChatTypes
+
+    #: ID of chat.
+    group_chat_id: UUID
+
+    #: users in chat.
+    members: List[UUID]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
```

Comment:
Are you sure? What if bot was added in group chat not at creation?

Reply:
I clarified in last commit